konghao/server/src/server.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// Created by andykong on 22-6-23.
+//
+#include "ros/ros.h"
+#include "std_msgs/Header.h"
+#include <client/Num.h>
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Found C system header after other header. Should be: server.h, c system, c++ system, other.  [build/include_order] [4]